src/main/java/frc/robot/subsystems/intake/BeamBreaker.java

```diff
@@ -0,0 +1,40 @@
+package frc.robot.subsystems.intake;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+import frc.robot.Ports;
+import frc.robot.subsystems.LoggedSubsystem;
+
+
+public class BeamBreaker extends LoggedSubsystem<BeamBreakerLoggedInputs> {
+    public DigitalInput beam = new DigitalInput(Ports.Intake.BEAM_BREAKER_SENSOR);
+    boolean isBlocking;
```

Comment:
Should be private

Reply:
Also, why do you have this variable if you don't use it

Reply:
I thought I might use it

Reply:
deleted.